dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/QuayImageRegistry.java

```diff
@@ -270,4 +270,45 @@ private void checkTriggers(Container container) {
             container.setValidTrigger(validTrigger);
         }
     }
+
+    /**
+     * Get the map of the given Quay container
+     * @param container
+     * @return
+         */
+    public Map<String, Object> getQuayInfo(final Container container){
+        final String repo = container.getNamespace() + '/' + container.getName();
+        final String repoUrl = QUAY_URL + "repository/" + repo;
+        final Optional<String> asStringBuilds = ResourceUtilities.asString(repoUrl, quayToken.getContent(), client);
+
+        if (asStringBuilds.isPresent()) {
+            final String json = asStringBuilds.get();
+
+            Gson gson = new Gson();
+            Map<String, Object> map = new HashMap<>();
+            map = (Map<String,Object>) gson.fromJson(json, map.getClass());
+            return map;
+
+        }
+        return null;
+    }
+
+    /**
+     * Get the map of organizations of the current user
+     * @return
+         */
+    public Map<String, ArrayList> getUserOrgs() {
```

Comment:
Could you give this a shot using the quay.io swagger client? (e.g. https://github.com/ga4gh/dockstore/blob/feature/only_register_if_owner_quay/dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/QuayImageRegistry.java#L94 ) 

Trying to deprecate the use of the non typesafe gson approach.